frontend/jupyter/cypress/e2e/main-page.cy.ts

```diff
@@ -8,7 +8,7 @@ describe('Main table', () => {
     });
     cy.fixture('notebooks').as('notebooksRequest');
     cy.fixture('pvcs').as('pvcsRequest');
-
+    const notebookInfoLoaded = true;
```

Comment:
Although this might fix failing tests, it is not how they should be fixed as this variable has nothing to do with the tests or any mock data given to tests, and defining a variable like this is most likely not best practices. That variable should be handled by the component it is used in. The tests are probably failing for another reason that we should investigate.